v3/plugins/redhat-developer/che-omnisharp-plugin/latest/meta.yaml

```diff
@@ -16,4 +16,4 @@ spec:
   - image: eclipse/che-remote-plugin-dotnet-2.2.105:next
     memoryLimit: "1024Mi"
   extensions:
-  - https://github.com/redhat-developer/omnisharp-theia-plugin/releases/download/v0.0.1/omnisharp_theia_plugin.theia
+  - https://github.com/redhat-developer/omnisharp-theia-plugin/releases/download/v0.0.2/omnisharp_theia_plugin.theia
```

Comment:
Please update `firstPublicationDate` to match the new latest version: `firstPublicationDate: "2019-06-19"`

Reply:
Updated